### ISSUE TITLE
NativeScript 3.0 compatibility

### ISSUE THREAD

**Issue by lookzovt** [burawi/tns-i18n#1]
Hello. Could you prepare tns-i18n for the current version of NativeScript? Thank you in advance.

**Comment by burawi** [comment]
Ok! we'll take a look on it.
thanks for reporting

**Comment by lookzovt** [comment]
Great, thank you. Your plugin is very useful.

**Comment by lookzovt** [comment]
If anybody is interested, you have to patch index.js file and change:
`    application.resources._L = _L;`
to:
`    application.setResources(_L);`

**Comment by burawi** [comment]
Thanks for the help! and I'm sorry for having took all this time without fixing it.

I did what you said. is it working now ? 

ps: the new version is (0.0.8)

**Comment by lookzovt** [comment]
You need commit changes to github. It's still 0.0.6.

**Comment by burawi** [comment]
done

**Comment by tkaarrela** [comment]
Hi,

0.0.8 does not work for me.

I'm getting 'Cannot find function or filter: _L' errors

**Comment by burawi** [comment]
Hi @tkaarrela 
I'm busy working on other projects... can you please find out the problem in the code and tell me what to fix ?

**Comment by tkaarrela** [comment]
Hi @burawi 

I managed to get plugin to work with ns 3 with following index.js

```
require('globals');
var application = require("application");
var platformModule = require("platform");
var strRender = require('str-render');

module.exports = function (defaultLang) {
  var lang = platformModule.device.language;
  var defaults = require('~/i18n/' + defaultLang);
  var strings = {};
  try {
    strings = require('~/i18n/' + lang);
  } catch (e) { }


  var _L = function (strName, ...replacers) {
    var res = '';
    if (strings.hasOwnProperty(strName))
      res = strings[strName];
    else if (defaults.hasOwnProperty(strName))
      res = defaults[strName];
    return strRender(res, '%s', ...replacers);
  }
  var applicationResources = application.getResources();
  applicationResources._L = _L;
  application.setResources(applicationResources);
  global._L = _L;
};
```

**Comment by burawi** [comment]
@tkaarrela Thanks man!
I just published it (0.0.9)

**Comment by radeva** [comment]
It looks like this issue was solved. Is it possible to close it?
